code/modules/mob/living/living.dm

```diff
@@ -494,13 +494,17 @@ default behaviour is:
 	if (buckled)
 		return
 
+	// Anything that observes us moving, try to activate.
+	for(var/mob/living/M in oviewers(world.view, src))
```

Comment:
note, this is alledgedly a buff to mob ai, as it makes have the same vision as a player (which was apparantly not the case before)?

Reply:
TLDR, mobs would make a list() for all other mobs on the zlevel it occupies, and would run through it to check if it should activate (Based on faction/distance from it)

This change means that anytime a living entity moves within a mob's vision (Mob A moves, mobs B sees A move, so List(B) is generated on mob A)